lib/sgetgrent.c

```diff
@@ -37,22 +37,16 @@ static char **list (char *s)
 	static char **members = NULL;
 	static int size = 0;	/* max members + 1 */
 	int i;
-	char **rbuf;
 
 	i = 0;
 	for (;;) {
 		/* check if there is room for another pointer (to a group
 		   member name, or terminating NULL).  */
 		if (i >= size) {
 			size = i + 100;	/* at least: i + 1 */
-			rbuf = realloc (members, size * sizeof (char *));
-			if (!rbuf) {
-				free (members);
-				members = NULL;
-				size = 0;
+			members = reallocf (members, size * sizeof (char *));
```

Comment:
Ah, yes, this is a good use of reallocf, as there's no resulting side effect to the caller.

Reply:
BTW, note for myself tomorrow, I need to say _before_ rather than _after_ in the commit message.  I was confused just now reading it myself (didn't remember what I meant).  :D

Reply:
Fixed.